doc/user_guide/dynamo0p3.rst

```diff
@@ -991,40 +991,26 @@ The list of rules for DoF Kernels is as follows:
    This rule reflects that a Kernel operates on some subset of the
    whole domain and is therefore designed to be called from within
    a loop that iterates over those subsets of the domain. Only fields
-   are accepted for a Kernel operating on DoFs because fields are the
-   only argument type that contain DoFs (i.e. field vectors are not allowed).
+   (as opposed to e.g. field vectors or operators) are accepted for DoF
+   Kernels because only they have a single value at each DoF.
 
-2) All Kernel arguments must be either fields or scalars (`real-`` and/or
+2) All Kernel arguments must be either fields or scalars (`real-` and/or
    `integer`-valued). DoF Kernels cannot accept operators.
 
 3) All field arguments to a given DoF Kernel must be on the same function
-   space. This is because all fields should have the same number.
-   It also means that PSyclone can determine the number of DoFs uniquely when
-   writing to a scalar.
+   space so they have the same number of DoFs.
 
-4) They must have at least one modified (i.e. writen to) argument. Unlike
+4) They must have at least one modified (i.e. writen to) field argument. Unlike
    built-ins, this is not limited and more than one modified argument is
    allowed.
 
 5) A Kernel may not write to a scalar argument. (Only built-ins are permitted
    to do this.) Any scalar arguments must therefore be declared in the metadata
    as `GH_READ` - see :ref:`below<dynamo0-3-kernel-valid-access>`
```

Comment:
This link target isn't right. Probably should be "dynamo0.3-kernel-valid-access"? You can check this by doing:

    cd PSyclone
    pip install -e .[doc]
    cd doc/user_guide
    make html

and then check for any errors or warnings.

Reply:
thank you :)